JesusFilm/core
# feat: remove use of deprecated events fields

# Description

Removing use of deprecated fields so they can be removed from the events data structure

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29689022/todos/5452435471)

# How should this PR be QA Tested?

All events should still trigger create mutations by successfully calling the createEvent in network tab of browser for the list of events.

1. Create/ use a journey that includes the blocks needed for the events
2. Preview the journey and trigger the event
3. Right click on the web page and click inspect
4. Navigate to the network tab, you should see a list of network requests
5. From the list of network requests, the ones that you are looking out for are `graphql-stage.jesusfilm.org` for staging environment and `graphql.jesusfilm.org` for the production environment
6. Select the network request for the event, and navigate to response tab of the request
7. Make sure there are no error messages in the response

Events:
- [x] ButtonClick (trigger by clicking a buttonBlock)
- [x] JourneyView (trigger by viewing any journey)
- [x] RadioQuestionSubmission (trigger by selecting a option on a radio question block)
- [x] SignUpSubmission (trigger by submitting a valid name and email on a signup block)
- [x] StepView (trigger by viewing a step)
- [x] TextResponseSubmission (trigger by submitting a valid response on a textResponse block)
- [x] VideoStart (trigger by starting a vide)
- [x] VideoPlay (trigger by pressing the play button)
- [x] VideoPause (trigger by pressing the pause button)
- [x] VideoComplete (trigger by watching the entirety of a video block)
- [x] VideoExpand (trigger by fullscreening the video player)
- [x] VideoCollapse (trigger by minimizing the fullscreen mode for the video player)
- [x] VideoProgress (triggers by watching the video to 25%, 50% and 75% of a video block)

# Checklist
